src/scc_hypervisor_collector/cli/__init__.py

```diff
@@ -59,19 +104,27 @@ def main(argv: Optional[Sequence[str]] = None) -> None:
     cfg_mgr = ConfigManager(config_file=args.config,
                             config_dir=args.config_dir,
                             check=args.check)
-
-    logging.info("ConfigManager: config_data = %s", repr(cfg_mgr.config_data))
+    try:
+        logger.info("ConfigManager: config_data = %s",
+                    repr(cfg_mgr.config_data))
+    except (ConfigManagerException, CollectorException) as e:
```

Comment:
So `ConfigManagerException` is derived from `CollectorException`, so you can just catch `CollectorException` here if needs be. ;-)

```suggestion
    except CollectorException as e:
```